mergify_cli/__init__.py

```diff
@@ -105,8 +105,7 @@ def get_slug(url: str) -> tuple[str, str]:
         path = parsed.path[1:].rstrip("/")
 
     user, repo = path.split("/", 1)
-    if repo.endswith(".git"):
-        repo = repo[:-4]
+    repo = repo.removesuffix(".git")
```

Comment:
just do this in a different pr instead of  all pr lol

Reply:
lol, It's not me, I guess my editor fix this crap alone 🤣